stable-25-1-4: ydb/tests/compatibility/test_compatibility.py.TestCompatibility.test_simple[restart_stable-25-1-4_to_current-column]

Test Report Information
Workflow Run: [17065953306](https://github.com/ydb-platform/ydb/actions/runs/17065953306)

Build Preset: relwithdebinfo

Branch: main

https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/17065953306/ya-main-x86-64/try_1/ya-test.html

ydb/tests/compatibility/test_compatibility.py.TestCompatibility.test_simple[restart_stable-25-1-4_to_current-column]

```
ydb/tests/compatibility/test_compatibility.py:87: in test_simple
    create_table(self, store_type)
ydb/tests/compatibility/test_compatibility.py:76: in create_table
    session.execute_scheme(
contrib/python/ydb/py3/ydb/table.py:1775: in execute_scheme
    return self._driver(
contrib/python/ydb/py3/ydb/tracing.py:70: in wrapper
    return f(self, *args, **kwargs)
contrib/python/ydb/py3/ydb/pool.py:464: in __call__
    res = connection(
contrib/python/ydb/py3/ydb/connection.py:469: in __call__
    return response if wrap_result is None else wrap_result(rpc_state, response, *wrap_args)
contrib/python/ydb/py3/ydb/_session_impl.py:20: in decorator
    return func(rpc_state, response_pb, session_state, *args, **kwargs)
contrib/python/ydb/py3/ydb/_session_impl.py:93: in wrap_execute_scheme_result
    issues._process_response(response_pb.operation)
contrib/python/ydb/py3/ydb/issues.py:237: in _process_response
    raise exc_obj(_format_response(response_proto), response_proto.issues)
E   ydb.issues.SchemeError: message: "Execution" issue_code: 1060 severity: 1 issues { position { row: 8 column: 67 } message: "Executing CREATE TABLE" end_position { row: 8 column: 67 } severity: 1 issues { message: "Type \'Decimal(22,9)\' specified for column \'income\', but support for parametrized decimal is disabled (EnableParameterizedDecimal feature flag is off)" issue_code: 2003 severity: 1 } } (server_code: 400070)

logsdir:
/home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/compatibility/test-results/py3test/testing_out_stuff
log:
/home/runner/actions_runner/_work/ydb/ydb/tmp/out/ydb/tests/compatibility/test-results/py3test/testing_out_stuff/test_compatibility.py.TestCompatibility.test_simple.restart_stable-25-1-4_to_current-column.log
```

--- a/ydb/tests/compatibility/test_compatibility.py
+++ b/ydb/tests/compatibility/test_compatibility.py
@@ -16,6 +16,7 @@ class TestCompatibility(RestartToAnotherVersionFixture):
         yield from self.setup_cluster(
             extra_feature_flags={
                 # "enable_table_datetime64": True # uncomment for 64 datetime in tpc-h/tpc-ds
+                "enable_decimal_type": True,
                 },
             column_shard_config={
                 'disabled_on_scheme_shard': False,

